apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/QueryOptionsTests.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright 2020 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Apis.Auth.OAuth2;
+using Google.Cloud.Spanner.Data.CommonTesting;
+using Google.Cloud.Spanner.V1;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Google.Cloud.Spanner.Data.IntegrationTests
+{
+    [Collection(nameof(ReadTableFixture))]
+    [CommonTestDiagnostics]
+    public class QueryOptionsTests
+    {
+        private readonly ReadTableFixture _fixture;
+
+        public QueryOptionsTests(ReadTableFixture fixture) =>
+            _fixture = fixture;
+
+        private async Task<T> ExecuteAsync<T>(string sql)
+        {
+            using (var connection = _fixture.GetConnection())
+            {
+                var cmd = connection.CreateSelectCommand(sql);
+                var result = await cmd.ExecuteScalarAsync<T>();
+                return result;
+            }
+        }
+
+        // [START spanner_test_single_key_read_with_query_options]
```

Comment:
It probably wouldn't hurt to add equivalent command-level option tests; I'll leave that as a call for you to make though.